tools/config_manager.py

```diff
@@ -81,6 +82,30 @@ def validate_config_file(config=None, schema_file=CONFIG_FILE_SCHEMA):
             return False, e
         return True, None
 
+    @staticmethod
+    def config_health_check(config):
+        # ensure api key encryption
+        should_replace_config = False
+        if CONFIG_EXCHANGES in config:
+            for exchange, exchange_config in config[CONFIG_EXCHANGES].items():
+                for key in CONFIG_EXCHANGE_ENCRYPTED_VALUES:
+                    try:
+                        if not ConfigManager._handle_encrypted_value(key, exchange_config, verbose=True):
```

Comment:
should_replace_config &= not ConfigManager._handle_encrypted_value(key, exchange_config, verbose=True)

Reply:
It would be 
```python
should_replace_config = (not ConfigManager._handle_encrypted_value(key, exchange_config, verbose=True)) and should_replace_config 
```
because in the other way around, when should_replace_config is False, python would not evaluate the second member of the & => therefore not encrypt other key

=> I think it's easier to understand, this way than in one line in this case, what is you opinion ?